include/ck_tile/ops/fmha/kernel/fmha_fwd_kernel.hpp

```diff
@@ -122,6 +139,15 @@ struct FmhaFwdKernel
         ck_tile::index_t nhead_stride_k;
         ck_tile::index_t nhead_stride_v;
         ck_tile::index_t nhead_stride_o;
+
+        QElementFunction q_element_func;
```

Comment:
can we just add the necessary ones for fp8 quantization? We don't know the performance imact of adding large number of objects in kernel argument.

Reply:
f7d81364f39fc3a9241846130518380e91ea48d2